client/src/components/Modals/SignUpModal.tsx

```diff
@@ -1,5 +1,16 @@
-// import Button from '@mui/material/Button';
 import React from 'react';
+import routes from '../../routes/routes';
+import Grid from '@mui/material/Grid';
```

Comment:
I'm wondering if it's possible to consolidate the MUI imports, like so ...
```
import {
  Grid,
  Dialog,
  Button,
  ...
} from '@mui/material';
```

Also from what I've seen in other projects (and we don't have to follow it), but it might be nice to arrange imports based on if the import is for a library or package, then followed by internal components, then images, then styling.  We just separate the set by a space, so for example ...

```
import React from 'react';
import { Link } 'react-router-dom';
import {
  Grid,
  Dialog,
  ...
} from '@mui/material';

import routes from '../../routes/routes';
```

 Let me know what you think.  Thanks.